container_workflow_tool/config/f31.yaml

```diff
@@ -42,8 +42,7 @@ v1:
   urls: !include share/urls.yaml
 
   commands:
-    2: "for ver in */; do make $ver/root/help.1; done"
```

Comment:
iirc Fedora OSBS did not generate the help files automatically (as we do internally for rhel7 builds) which is why this had to be added. Can you check if this is still the case?

Reply:
The cycle `for ver in */; ` is wrong, It takes also examples, I will try to review it a bit so the results are only digits.
 like:
`for ver in [0-9]*/;`

I will test it.

Reply:
Well, I have test my proposal and it is working.

```bash
[root@0ff1f6026406 /]# /tmp/run.sh
+ '[' -z /tmp/betka-generator/20200518102030-s2i-python-container/ ']'
+ '[' -z python3 ']'
+ '[' -z s2i-python-container ']'
+ SOURCES=/tmp/betka-generator/20200518102030-s2i-python-container/
+ RESULTS=/tmp/betka-generator/20200518102030-s2i-python-container//results
+ CWT_WD_PREFIX=sync
+ CWT_WD=/tmp/sync
+ CWT_CONFIG_DIR=/tmp/container-workflow-tool/container_workflow_tool/config
+ cd /tmp/betka-generator/20200518102030-s2i-python-container//results
++ git rev-parse --abbrev-ref HEAD
+ config=f31
+ echo f31
+ grep -q master
+ cd /tmp/container-workflow-tool
+ read -r _ ups_name _
++ cwt --config /tmp/container-workflow-tool/container_workflow_tool/config/f31.yaml --do-image python3 utils listupstream
+ mkdir -p /tmp/sync/upstreams/
+ cp -r /tmp/betka-generator/20200518102030-s2i-python-container//s2i-python-container /tmp/sync/upstreams
+ mv /tmp/sync/upstreams/s2i-python-container /tmp/sync/upstreams/python3
+ cp -r /tmp/betka-generator/20200518102030-s2i-python-container//results /tmp/sync
+ mv /tmp/sync/results /tmp/sync/python3
+ cwt -v 5 --config /tmp/container-workflow-tool/container_workflow_tool/config/f31.yaml --disable-klist --base sync --do-image python3 git --commit-msg '' pullupstream
Using working directory: /tmp/sync
Using existing downstream repo: python3
container_workflow_tool.main.dist-git - DEBUG: No release information found in Dockerfile
Using existing repository.
container_workflow_tool.main.dist-git - DEBUG: Running commands in upstream repo.
container_workflow_tool.main.dist-git - DEBUG: Running '1' command 'make generate-all'
container_workflow_tool.main.dist-git - DEBUG: Running '2' command 'for ver in [0-9]*/; do make $ver/root/help.1; done'
container_workflow_tool.main.dist-git - DEBUG: Running '3' command 'sed -i 's|registry.fedoraproject.org/f[0-9]*/|registry.fedoraproject.org/f31/|' */Dockerfile.fedora'
container_workflow_tool.main.dist-git - DEBUG: rmtree python3/test
container_workflow_tool.main.dist-git - DEBUG: cp -r upstreams/python3/3.7/test python3/test
container_workflow_tool.main.dist-git - DEBUG: cp upstreams/python3/3.7/README.md python3/README.md
container_workflow_tool.main.dist-git - DEBUG: rmtree python3/s2i
container_workflow_tool.main.dist-git - DEBUG: cp -r upstreams/python3/3.7/s2i python3/s2i
container_workflow_tool.main.dist-git - DEBUG: rmtree python3/root
container_workflow_tool.main.dist-git - DEBUG: cp -r upstreams/python3/3.7/root python3/root
container_workflow_tool.main.dist-git - DEBUG: cp upstreams/python3/3.7/cccp.yml python3/cccp.yml
container_workflow_tool.main.dist-git - DEBUG: cp upstreams/python3/3.7/Dockerfile.fedora python3/Dockerfile.fedora
container_workflow_tool.main.dist-git - DEBUG: looking for dangling symlink python3/test/micropipenv-requirements-test-app (that points to python3/test/../../examples/micropipenv-requirements-test-app)
container_workflow_tool.main.dist-git - DEBUG: unlink python3/test/micropipenv-requirements-test-app
container_workflow_tool.main.dist-git - DEBUG: cp -r upstreams/python3/3.7/test/../../examples/micropipenv-requirements-test-app python3/test/micropipenv-requirements-test-app
container_workflow_tool.main.dist-git - DEBUG: looking for dangling symlink python3/test/pipenv-and-micropipenv-should-fail-test-app (that points to python3/test/../../src/test/pipenv-and-micropipenv-should-fail-test-app)
container_workflow_tool.main.dist-git - DEBUG: unlink python3/test/pipenv-and-micropipenv-should-fail-test-app
container_workflow_tool.main.dist-git - DEBUG: cp -r upstreams/python3/3.7/test/../../src/test/pipenv-and-micropipenv-should-fail-test-app python3/test/pipenv-and-micropipenv-should-fail-test-app
container_workflow_tool.main.dist-git - DEBUG: looking for dangling symlink python3/test/mod-wsgi-test-app (that points to python3/test/../../examples/mod-wsgi-test-app)
container_workflow_tool.main.dist-git - DEBUG: unlink python3/test/mod-wsgi-test-app
container_workflow_tool.main.dist-git - DEBUG: cp -r upstreams/python3/3.7/test/../../examples/mod-wsgi-test-app python3/test/mod-wsgi-test-app
container_workflow_tool.main.dist-git - DEBUG: looking for dangling symlink python3/test/setup-requirements-test-app (that points to python3/test/../../examples/setup-requirements-test-app)
container_workflow_tool.main.dist-git - DEBUG: unlink python3/test/setup-requirements-test-app
container_workflow_tool.main.dist-git - DEBUG: cp -r upstreams/python3/3.7/test/../../examples/setup-requirements-test-app python3/test/setup-requirements-test-app
container_workflow_tool.main.dist-git - DEBUG: looking for dangling symlink python3/test/test-lib-openshift.sh (that points to python3/test/../../common/test-lib-openshift.sh)
container_workflow_tool.main.dist-git - DEBUG: unlink python3/test/test-lib-openshift.sh
container_workflow_tool.main.dist-git - DEBUG: cp upstreams/python3/3.7/test/../../common/test-lib-openshift.sh python3/test/test-lib-openshift.sh
container_workflow_tool.main.dist-git - DEBUG: looking for dangling symlink python3/test/run-openshift (that points to python3/test/../../test/run-openshift)
container_workflow_tool.main.dist-git - DEBUG: unlink python3/test/run-openshift
container_workflow_tool.main.dist-git - DEBUG: cp upstreams/python3/3.7/test/../../test/run-openshift python3/test/run-openshift
container_workflow_tool.main.dist-git - DEBUG: looking for dangling symlink python3/test/test-lib.sh (that points to python3/test/../../common/test-lib.sh)
container_workflow_tool.main.dist-git - DEBUG: unlink python3/test/test-lib.sh
container_workflow_tool.main.dist-git - DEBUG: cp upstreams/python3/3.7/test/../../common/test-lib.sh python3/test/test-lib.sh
container_workflow_tool.main.dist-git - DEBUG: looking for dangling symlink python3/test/locale-test-app (that points to python3/test/../../examples/locale-test-app)
container_workflow_tool.main.dist-git - DEBUG: unlink python3/test/locale-test-app
container_workflow_tool.main.dist-git - DEBUG: cp -r upstreams/python3/3.7/test/../../examples/locale-test-app python3/test/locale-test-app
container_workflow_tool.main.dist-git - DEBUG: looking for dangling symlink python3/test/app-home-test-app (that points to python3/test/../../examples/app-home-test-app)
container_workflow_tool.main.dist-git - DEBUG: unlink python3/test/app-home-test-app
container_workflow_tool.main.dist-git - DEBUG: cp -r upstreams/python3/3.7/test/../../examples/app-home-test-app python3/test/app-home-test-app
container_workflow_tool.main.dist-git - DEBUG: looking for dangling symlink python3/test/django-test-app (that points to python3/test/../../examples/django-test-app)
container_workflow_tool.main.dist-git - DEBUG: unlink python3/test/django-test-app
container_workflow_tool.main.dist-git - DEBUG: cp -r upstreams/python3/3.7/test/../../examples/django-test-app python3/test/django-test-app
container_workflow_tool.main.dist-git - DEBUG: looking for dangling symlink python3/test/numpy-test-app (that points to python3/test/../../examples/numpy-test-app)
container_workflow_tool.main.dist-git - DEBUG: unlink python3/test/numpy-test-app
container_workflow_tool.main.dist-git - DEBUG: cp -r upstreams/python3/3.7/test/../../examples/numpy-test-app python3/test/numpy-test-app
container_workflow_tool.main.dist-git - DEBUG: looking for dangling symlink python3/test/standalone-test-app (that points to python3/test/../../examples/standalone-test-app)
container_workflow_tool.main.dist-git - DEBUG: unlink python3/test/standalone-test-app
container_workflow_tool.main.dist-git - DEBUG: cp -r upstreams/python3/3.7/test/../../examples/standalone-test-app python3/test/standalone-test-app
container_workflow_tool.main.dist-git - DEBUG: looking for dangling symlink python3/test/setup-test-app (that points to python3/test/../../examples/setup-test-app)
container_workflow_tool.main.dist-git - DEBUG: unlink python3/test/setup-test-app
container_workflow_tool.main.dist-git - DEBUG: cp -r upstreams/python3/3.7/test/../../examples/setup-test-app python3/test/setup-test-app
container_workflow_tool.main.dist-git - DEBUG: looking for dangling symlink python3/test/npm-virtualenv-uwsgi-test-app (that points to python3/test/../../examples/npm-virtualenv-uwsgi-test-app)
container_workflow_tool.main.dist-git - DEBUG: unlink python3/test/npm-virtualenv-uwsgi-test-app
container_workflow_tool.main.dist-git - DEBUG: cp -r upstreams/python3/3.7/test/../../examples/npm-virtualenv-uwsgi-test-app python3/test/npm-virtualenv-uwsgi-test-app
container_workflow_tool.main.dist-git - DEBUG: Removing changes for: .gitignore
container_workflow_tool.main.dist-git - DEBUG: Removing changes for: Dockerfile.rhel7
container_workflow_tool.main.dist-git - DEBUG: Removing changes for: Dockerfile.rhel8
container_workflow_tool.main.dist-git - DEBUG: Removing changes for: .osbs-repo-config
container_workflow_tool.main.dist-git - DEBUG: Removing changes for: .cccp.yml
container_workflow_tool.main.dist-git - DEBUG: Removing changes for: cccp.yml
container_workflow_tool.main.dist-git - DEBUG: Removing untracked ignored file: cccp.yml
container_workflow_tool.main.dist-git - DEBUG: Removing changes for: content_sets.yml
Not creating new commit in: python3

Git location: /tmp/sync
You can view changes made by running:
./rebuild-helper --base sync git show
To push and build run: rebuild-helper git push && rebuild-helper build [base/core/s2i] --repo-url link-to-repo-file
+ cp -r /tmp/sync/python3/3.7 /tmp/sync/python3/Dockerfile /tmp/sync/python3/Dockerfile.fedora /tmp/sync/python3/README.md /tmp/sync/python3/help.md /tmp/sync/python3/root /tmp/sync/python3/s2i /tmp/sync/python3/test /tmp/betka-generator/20200518102030-s2i-python-container//results
```

I will update PR.

Reply:
It should be fixed by the latest commit.